src/model/Ingredient.ts

```diff
@@ -1,34 +1,28 @@
 import { Field, ID, ObjectType } from "type-graphql";
-import { Unit } from "./Unit";
 
 @ObjectType({ description: "A single ingredient for a recipe" })
 export class Ingredient {
   @Field(ID)
-  id: number;
+  id: string;
 
   @Field()
   name: string;
 
   @Field()
-  unit: Unit;
```

Comment:
Wieso fliegt Unit jetzt raus?

Reply:
Weil das vom Rezept abhängt. Es könnte theoretisch sein, dass ein Rezept die Menge an Kartoffeln in Gramm und ein anderes in Stück angibt.

Reply:
Alternativ könnte ich es drin lassen und optional machen, falls es später für die Rezepte so sinnvoller ist. Aber das hätte ich sonst auf später verschoben, wenn die Rezepte dann implementiert werden.

Reply:
ah ok. das ergibt natürlich sinn.
Passt so